ddlite.py

```diff
@@ -358,28 +426,134 @@ def apply_labelers(self, labelers_f, clear=False):
     """
     nr_old = self.num_labelers() if not clear else 0
     add = sparse.lil_matrix((self.num_candidates(), len(labelers_f)))
-    self.labelers = add if (self.labelers is None or clear)\
-                        else sparse.hstack([self.labelers,add], format = 'lil')
+    add_names = [lab.__name__ for lab in labelers_f]
+    if self.labelers is None or clear:
+      self.labelers = add
+      self.labeler_names = add_names
+    else:
+      self.labelers = sparse.hstack([self.labelers,add], format = 'lil')
+      self.labeler_names.extend(add_names)
     for i,c in enumerate(self.C._candidates):    
       for j,labeler in enumerate(labelers_f):
         self.labelers[i,j + nr_old] = labeler(c)
-        
-  def learn_weights(self, nSteps=1000, sample=False, nSamples=100, mu=1e-9, 
-                    holdout=0.1, use_sparse = True, verbose=False):
+    
+  def _coverage(self):
+    cov = [(self.labelers == lab).sum(1) for lab in [1,-1]]
+    abst = self.num_labelers() - cov[0] - cov[1]
```

Comment:
It does refer to the number of labeling functions (#abstain for given candidate = #labelers - #pos - #neg)